src/components/screens/consul/Proposals/ProposalDetail.js

```diff
@@ -149,7 +149,8 @@ export const ProposalDetail = ({ data, refetch, route, navigation }) => {
                   description,
                   documents,
                   id,
-                  image: imageUrlMedium,
+                  image: image?.imageUrlLarge,
+                  imageId: image?.id,
```

Comment:
what is `imageId` used for? i can only find `image: data?.image || '',` in `NewProposal`.

Reply:
The `imageId` information is required to delete the image. But yes you are right, this information is also in the `image` object.

Reply:
ok right and we need it concerning deletions in the other PR, i see